Riot/Modules/Room/RoomViewController.m

```diff
@@ -956,14 +958,22 @@ - (void)stopActivityIndicator
         notificationTaskProfile = nil;
     }
     if ([self providesCustomActivityIndicator]) {
+        // The legacy super implementation of `stopActivityIndicator` contains a number of checks grouped under `canStopActivityIndicator`
+        // to determine whether the indicator can be stopped or not (and the method should thus rather be called `stopActivityIndicatorIfPossible`).
+        // Since the newer indicators are not calling super implementation, the check for `canStopActivityIndicator` has to be performed manually.
         if ([self canStopActivityIndicator]) {
-            [self.delegate roomViewControllerDidStopLoading:self];
+            [self stopLoadingIndicator];
         }
     } else {
         [super stopActivityIndicator];
     }
 }
 
+- (void)stopLoadingIndicator
```

Comment:
As a pending of `presentActivityIndicator` shouldn't we call this method `stopActivityIndicator`?